### ISSUE TITLE
Add status overview for lit-html and lit-element components

### ISSUE THREAD

**Issue by MarcusMorba** [vaadin/vaadin#49]
It would be great to get an overview of the status which components have been (re-)developed already based on lit-html and lit-element on this page: https://github.com/vaadin/vaadin

Thanks.

**Comment by MarcusMorba** [comment]
@tomivirkki - is it correct, that also with v15 all components still are based on polymer3?

**Comment by brion-fuller** [comment]
Is there a roadmap for lit-element?  From what I understand lit-element is not a production ready product.  Other products like hyperHTML are more prepped for production.

**Comment by web-padawan** [comment]
We do not have a public roadmap yet, but here is a summary of our current effort:
https://vaadin.com/blog/next-generation-vaadin-components

TL;DR: `vaadin-details` 2.0.0-alpha1 is out. In the upcoming weeks, I'm going to work on the   vaadin/vaadin-accordion#28 and vaadin/vaadin-tabs#142

We will let you know once we have a more clear plan for components rewrite in 2020.
